package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "next lint",
     "storybook": "start-storybook -p 6006",
     "storybook:build": "build-storybook",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "prepare": "husky install"
```

Comment:
on this branch with on a clean install, this step fails 👀 

you may need to bump up the version of husky to resolve this

Reply:
Yes I just did.